app/models/tenant.rb

```diff
@@ -99,6 +99,33 @@ def login_logo_content_type
     end
   end
 
+  def get_quotas
```

Comment:
I think my big problem with these methods is these are for presentation.  Maybe we can move the get and set to a presenter thing that converts AR objects to whatever format the UI needs right now.  I don't think it needs to happen here though.